### PR TITLE
DO-245 UX  Enhancement of Sorting in desktop

### PR DESCRIPTION
**Pull request by Palakagrawal28**

### CODE REVIEW

**Comment by mohitsaladiya** [comment]
Why this file pyc-collagee.png  is committed in this PR?

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/SelectedSort/SelectedSort.jsx [hunk 1]
```diff
@@ -51,15 +51,15 @@ const SelectedSort = (props) => {
 
   if (sort && sort.length) {
     return (
-      <div className="selected_sort_container" >
-        <span className="selected_sort_container__cancelButton" onClick={onClickClearSort}>
-          <Icon icon={faTimesCircle} />
+      <div className="selected_sort_container">
+        <span className="selected_sort_container__carrot" onClick={onClickCaret}>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/SelectedSort/SelectedSort.jsx:55]: carrot seriously?? 
```suggestion
        <span className="selected_sort__container__caret" onClick={onClickCaret}>
```

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/SelectedSort/SelectedSort.jsx [hunk 2]
```diff
@@ -51,15 +51,15 @@ const SelectedSort = (props) => {
...
+          {renderCaret()}
         </span>
         <span className="selected_sort_container__text">
           Sort By: {fieldLabels[sort] || sort}
         </span>
-        <span className="selected_sort_container__carrot" onClick={onClickCaret}>
-          {renderCaret()}
+        <span className="selected_sort_container__cancelButton" onClick={onClickClearSort}>
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/SelectedSort/SelectedSort.jsx:61]:
```suggestion
        <span className="selected_sort__container__cancelButton" onClick={onClickClearSort}>
```

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/SelectedSort/SelectedSort.jsx [hunk 3]
```diff
@@ -51,15 +51,15 @@ const SelectedSort = (props) => {
 
   if (sort && sort.length) {
     return (
-      <div className="selected_sort_container" >
-        <span className="selected_sort_container__cancelButton" onClick={onClickClearSort}>
-          <Icon icon={faTimesCircle} />
+      <div className="selected_sort_container">
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/SelectedSort/SelectedSort.jsx:54]:
```suggestion
      <div className="selected_sort__container">
```

Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/SelectedSort/SelectedSort.jsx:54]: Done

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/SelectedSort/SelectedSort.jsx (hunk 2 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/SelectedSort/SelectedSort.jsx:61]: Done

File: src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/SelectedSort/SelectedSort.jsx (hunk 1 above)
Comment (Palakagrawal28) [reply src/components/routes/Member/AuthenticatedMember/MemberListViewRoute/MemberList/SelectedSort/SelectedSort.jsx:55]: Done

**Pull request merged**